include/LightGBM/config.h

```diff
@@ -127,6 +127,7 @@ struct Config {
   // descl2 = label should be ``int`` type in lambdarank tasks, and larger number represents the higher relevance (e.g. 0:bad, 1:fair, 2:good, 3:perfect)
   // descl2 = `label_gain <#objective-parameters>`__ can be used to set the gain (weight) of ``int`` label
   // descl2 = all values in ``label`` must be smaller than number of elements in ``label_gain``
+  // desc = ``rank_xendcg``, `XE_NDCG_MART <https://arxiv.org/abs/1911.09798>`__ ranking objective function.
```

Comment:
Can we have some aliases, e.g. `xe_ndcg_mart`, `cross_entropy_lambdamart`, `xe_lambdamart`, `xe_ndcg`, etc. (you know better which aliases may help users to find and remember this objective)?

Reply:
That makes sense. I've added a few aliases.